org-member/policies/ecr-policy.json

```
@@ -0,0 +1,95 @@
+{
+  "rules": [
+    {
+      "rulePriority": 1,
+      "description": "Expire images with tagPrefix 'branch-master'",
+      "selection": {
+        "tagStatus": "tagged",
+        "tagPrefixList": ["branch-master"],
+        "countType": "ImageCountMoreThan",
+        "countNumber": 1
+      },
+      "action": {
+        "type": "expire"
+      }
+    },
+    {
+      "rulePriority": 2,
+      "description": "Expire images with tagPrefix 'branch-main'",
+      "selection": {
+        "tagStatus": "tagged",
+        "tagPrefixList": ["branch-main"],
+        "countType": "ImageCountMoreThan",
+        "countNumber": 1
+      },
+      "action": {
+        "type": "expire"
+      }
+    },
+    {
+      "rulePriority": 3,
+      "description": "Expire images with tagPrefix 'tag-latest'",
+      "selection": {
+        "tagStatus": "tagged",
+        "tagPrefixList": ["tag-latest"],
+        "countType": "ImageCountMoreThan",
+        "countNumber": 1
+      },
+      "action": {
+        "type": "expire"
+      }
+    },
+    {
+      "rulePriority": 4,
+      "description": "Expire images with tagPrefix 'latest'",
+      "selection": {
+        "tagStatus": "tagged",
+        "tagPrefixList": ["latest"],
+        "countType": "ImageCountMoreThan",
+        "countNumber": 1
+      },
+      "action": {
+        "type": "expire"
+      }
+    },
+    {
+      "rulePriority": 5,
+      "description": "Expire images with tagWildcard 'branch-*'",
+      "selection": {
+        "tagStatus": "tagged",
+        "tagPatternList": ["branch-*"],
+        "countType": "ImageCountMoreThan",
+        "countNumber": 3
```

We could have more than three things in play that involve images built of a branch for testing.

E.g. Team of 8 developers, even with pairing they could have 4 on the go quite easily.

This might be OK...

```suggestion
        "countType": "sinceImagePushed",
        "countUnit": "days",
        "countNumber": 30
```

Long lived branches are bad right ;-)